pom.xml

```diff
@@ -8,14 +8,14 @@
 		<cctimestamp>NO_TIMESTAMP</cctimestamp>
 		<scm.revision>NO_SCM_REVISION</scm.revision>
 
-		<version.proguard>7.0.0</version.proguard>
+		<version.proguard>7.1.0-beta3</version.proguard>
 	</properties>
 
 
 	<groupId>com.github.wvengen</groupId>
 	<artifactId>proguard-maven-plugin</artifactId>
 	<name>proguard-maven-plugin</name>
-	<version>2.3.1</version>
+	<version>2.3.2-beta</version>
```

Comment:
I should have done this myself, but it should be
2.3.2-SNAPSHOT like done in https://github.com/wvengen/proguard-maven-plugin/commit/5357b6621b56df86131805bf0121df5011425de5#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8

Reply:
Done.